base/centos7/Dockerfile

```diff
@@ -2,36 +2,12 @@ FROM centos:centos7
 LABEL maintainer="RStudio Docker <docker@rstudio.com>"
 
 RUN yum -y update && \
-    yum -y groupinstall "Development tools" && \
     yum -y install \
-    bzip2-devel \
-    gcc \
-    gcc-gfortran \
-    libcurl-devel \
-    libicu-devel \
-    libSM \
-    libtiff \
-    libXmu \
-    libXt \
-    make \
-    pango \
-    pcre-devel \
-    pcre2-devel \
+    fontconfig \
     perl-Digest-MD5 \
```

Comment:
I kept perl-Digest-MD5 since TinyTeX needed it (and also added fontconfig for xetex). Do you remember if perl-Digest-MD5 was an R dependency, or specifically for TinyTeX? We included it in the R rpm, but maybe it's not really required.

Reply:
I'm afraid I don't remember.